### DISCUSSION TITLE
setup is unable to acces the manager

### DISCUSSION THREAD

**Discussion by mohasmr** [Security-Onion-Solutions/securityonion#12465] [Unsupported Versions]
i have a problem i cant even identify the causes when i install a desktop to linked to a distributed manager and try to join it to an existing grid i have problem of connectivity 
manager ip:192.168.10.10
desktop ip : 192.168.10.90 
they are on the same network , in the manager console i puted the right configuration
![Capture](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/83bcb0c1-fdf3-4803-bf84-99e7d8b4eca3)
even i try with **so-firewall-minion --role=DESKTOP --ip=192.168.10.90**  there is no changes
i even tryied to shutdow the firewalld and droped all iptables but nothing 
the two security onion(desktop and manager) cant even ping each others 
please to help im stuck :'(

**Comment by cm-ops** [comment]
Is your SOD a VM? What about your manager?

**Comment by mohasmr** [comment]
Yes the two are vms on vmware workstation pro

Le ven. 1 mars 2024 à 19:05, Chris Morgret ***@***.***> a
écrit :

**Comment by mohasmr** [comment]
Any help about this issue

**Reply by cm-ops** [reply]
What are your network settings in VMware?

**Reply by mohasmr** [reply]
They are on bridge to Ethernet card because i have two physically machine each one have security-onion on her vmware so they are physically separated

The problem here is that every security onion can see the windows that hosting vmware but they can't see each others

**Reply by cm-ops** [reply]
What are you network settingsfor the bridged VMnet? Under `Edit > Virtual Network Editor`

**Reply by mohasmr** [reply]
![error](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/6cae700a-482c-409f-ba7c-5c46fa0dfe5d)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/c7d6776c-4daf-4447-9141-bde3c65c8123)

**Comment by mohasmr** [comment]
the issues caused by a missconfiguration on the switch the  so-firewall-minion --role=DESKTOP --ip=192.168.10.90 working correctly
